msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -492,6 +492,219 @@ The following image shows the default view of the Adaptive Card when you have no
 
 :::image type="content" source="../../assets/images/Cards/small-width-adaptive-card.png" alt-text="Small width Adaptive Card view":::
 
+### Adaptive Card responsive layout
+
+Adaptive Cards usually match the look and feel of the host application and have a fixed layout that remains consistent across Teams mobile and desktop clients.
+
+Adaptive Card responsive layout helps you to design Adaptive Cards with different layouts that adapt to different card widths in the context where the card is displayed. When you design Adaptive Card for your app, ensure that you provide a better user experience across chat, channels, and meeting chat in Teams desktop and mobile clients.
+
+[Add image for responsive Adaptive card in mobile and desktop.]
+
+#### Design responsive Adaptive Cards
+
+You can design a responsive Adaptive Card using the `targetWidth` property, which allows you to show or hide any element based on the card's width.
+
+You can set the `targetWidth` property to any element in your Adaptive Card to create different layouts for different card widths. The following table provides the available values of the `targetWidth` property for different layouts:
+
+  |Value  |Description  |
+  |---------|---------|
```

Comment:
|---------|---------|
  | `veryNarrow` | The element will be visible when the card is "very narrow", which typically is the case in a meeting chat. |
  | `narrow` | The element will be visible when the card is "narrow", which is typically the case on a mobile phone in portrait mode. |
  | `standard` | The element will be visible when the card's width is "standard", which is typically the case on a mobile phone in landscape mode, on a tablet in portrait mode, or in a chat on desktop. |
  | `wide` | The element will be visible when the card is "wide", which is typically the case on a tablet in landscape mode, in a channel on desktop, or in a chat on desktop when yoiu set your card to be "full width" (PLEASE INCLUDE LINK TO FULL WIDTH SECTION ON THIS VERY PAGE). |

Sometimes, you may want an element to target not just one specific card width but a range of card widths. For example, you might want an element to be visible when the card's width is "standard or above", or when the card's width is "narrow or below".

To achieve that, you can use the `atLeast` and atMost` modifier prefixes:
| Example | Description |
| --- | --- |
| `"targetWidth": "atLeast:standard"` | Will cause the element to be visible when the card's width is "at least standard", so "standard" or "wide" |
| `"targetWidth": "atMost:narrow"` | Will cause the element to be visible when the card's width is "at most narrow", so "very narrow" or "narrow" |

**Tip**: You do not have to set the `targetWidth` on all elements. When you do not set it on an element, that element will simply always be visible regardless of the card's width, which can be perfectly appropriate.

Reply:
updated